cli/cmd/proxy/loglevel/command.go

```diff
@@ -159,6 +172,9 @@ func (l *LogLevelCommand) parseFlags(args []string) error {
 }
 
 func (l *LogLevelCommand) validateFlags() error {
+	if l.level != "" && l.reset == true {
+		return errors.New("cannot set log levels and reset at the same time")
```

Comment:
I think it would be good to see the value the user tried to set the log level to in the error so they can grok what they need to change:

```suggestion
		return fmt.Errorf("cannot set log level to '%s' and reset to 'info' at the same time", l.level)
```

Separately, what do you think about changing this conditional to allowing the following _incorrect_, but still understandable input:

```
if (l.level != "" || l.level != "info") && l.reset {
```

It's just a little more forgiving to a mistaken and well-intentioned input.

Reply:
I like the change to error message, I'm a little hesitant on the conditional change, only because I could see it being confusing if 
`-l info -r` worked, but `-l warning -r` didn't, it feels like you've gotta know a bit more about the internals to understand why one is correct and the other is not

Reply:
That makes sense! I follow your logic and trust your judgement on this one